packages/office-ui-fabric-react/src/components/DetailsList/DetailsRow.styles.ts

```diff
@@ -0,0 +1,426 @@
+    defaultBackgroundColor: white,
+
+    // Hover
+    hoverTextColor: neutralPrimary,
+    hoverMetaTextColor: neutralSecondary,
+    hoverColorBackground: neutralLighter,
+
+    // Selected
+    selectedTextColor: neutralDark,
+    selectedMetaTextColor: neutralPrimary,
+    selectedBackgroundColor: neutralLight,
+
+    // Selected Hover
+    selectedHoverTextColor: black,
+    selectedHoverMetaTextColor: neutralDark,
+    selectedHoverBackgroundColor: neutralQuaternaryAlt,
+
+    // Focus
+    focusHeaderTextColor: black,
+    focusBackgroundColor: neutralQuaternaryAlt,
+    focusMetaTextColor: neutralDark
+  };
+
+  const thickBorderStyle = `${values.rowShimmerHorizontalBorder * 4}px solid ${colors.defaultBackgroundColor}`;
+  const thinBorderStyle = `${values.rowShimmerHorizontalBorder}px solid ${colors.defaultBackgroundColor}`;
+  const selectedStyles: IStyleBaseArray = [
+    getFocusStyle(theme, 0, undefined, undefined, themePrimary),
+    classNames.isSelected,
+    checkClassNames && checkClassNames.isSelected,
+    {
+      color: colors.selectedMetaTextColor,
+      background: colors.selectedBackgroundColor,
+      borderBottom: `1px solid ${white}`,
+      selectors: {
+        '&:before': {
+          position: 'absolute',
+          display: 'block',
+          top: -1,
+          height: 1,
+          bottom: 0,
+          left: 0,
+          right: 0,
+          content: '',
+          borderTop: `1px solid ${white}`
+        },
+
+        // Selected State hover
+        '&:hover': {
+          background: colors.selectedHoverBackgroundColor,
+          selectors: {
+            // Selected State hover meta cell
+            $cell: {
+              color: colors.selectedHoverMetaTextColor,
+
+              selectors: {
+                [HighContrastSelector]: {
+                  color: 'HighlightText',
+                  selectors: {
+                    '> a': {
+                      color: 'HighlightText'
+                    }
+                  }
+                },
+
+                // Selected State hover Header cell
+                '&.$isRowHeader': {
+                  color: colors.selectedHoverTextColor,
+                  selectors: {
+                    [HighContrastSelector]: {
+                      color: 'HighlightText'
+                    }
+                  }
+                }
+              }
+            }
+          }
+        },
+
+        // Focus state
+        '&:focus': {
+          background: colors.focusBackgroundColor,
+
+          selectors: {
+            // Selected State hover meta cell
+            $cell: {
+              color: colors.focusMetaTextColor,
+              [HighContrastSelector]: {
+                color: 'HighlightText',
+                selectors: {
+                  '> a': {
+                    color: 'HighlightText'
+                  }
+                }
+              },
+
+              // Row header cell
+              '&.$isRowHeader': {
```

Comment:
Have you verified the `$` references actually work? I seemed to have trouble with them...